### PR TITLE
[Java/en] Updated to include standard input using Scanner and System.in

### PR DESCRIPTION
**Pull request by seannam**
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
 - [x] Yes, I have double-checked quotes and field names!

--------
Requested here - issue #2309 
--------
Still a newbie when it comes to contributing to open source projects. Please let me know if there is anything I can do differently to improve my contributions!

### CODE REVIEW

File: java.html.markdown [hunk 1]
```diff
@@ -51,6 +52,14 @@ public class LearnJava {
     // point.
     public static void main (String[] args) {
 
+    	///////////////////////////////////////
+		// Input/Output
+		///////////////////////////////////////
```
Comment (adambard) [inline java.html.markdown]: Can you change the indentation to use 4 spaces? Stuff like the above will show up on the website too.

Comment (seannam) [inline java.html.markdown]: Will do (soon)

Comment (seannam) [inline java.html.markdown]: Should pull from master before making changes to my local version and submitting another pr?

Comment (vendethiel) [inline java.html.markdown]: please pull from master

**Comment by seannam** [comment]
I pulled from master and pushed it again.

File: java.html.markdown [hunk 2]
```diff
@@ -65,6 +74,32 @@ public class LearnJava {
         // Use System.out.printf() for easy formatted printing.
         System.out.printf("pi = %.5f", Math.PI); // => pi = 3.14159
 
+        /*
+        * Input
+        */
+
+		// use Scanner to read input
```
Comment (vendethiel) [inline java.html.markdown]: seems like the identation is still off :(

Comment (seannam) [inline java.html.markdown]: Tried to fix it again

File: java.html.markdown [hunk 3]
```diff
@@ -51,6 +52,14 @@ public class LearnJava {
     // point.
     public static void main (String[] args) {
 
+    ///////////////////////////////////////
+    // Input/Output
+    ///////////////////////////////////////
+
+		/*
```
Comment (vendethiel) [inline java.html.markdown]: indentation is still broken it seems. make sure you're not mixing up tabs/spaces.

Comment (seannam) [inline java.html.markdown]: I don't know what I'm doing wrong :(
I'll take another look during my break next week.

Comment (seannam) [inline java.html.markdown]: It's line 56 and 59, correct?

**Comment by adambard** [comment]
`:%s/\t/    /g`

Vim wins again!

**Comment by seannam** [comment]
vim! Now why didn't I think of that...
Thanks for getting this fixed.

**Pull request merged**